pdns/rec_channel_rec.cc

```diff
@@ -668,12 +668,21 @@ static string getTAs()
 template<typename T>
 string setMinimumTTL(T begin, T end)
 {
-  if(end-begin != 1) 
+  if(end-begin != 1)
     return "Need to supply new minimum TTL number\n";
   SyncRes::s_minimumTTL = pdns_stou(*begin);
   return "New minimum TTL: " + std::to_string(SyncRes::s_minimumTTL) + "\n";
 }
 
+template<typename T>
+string setMinimumECSTTL(T begin, T end)
+{
+  if(end-begin != 1)
+    return "Need to supply new ECS minimum TTL number\n";
+  SyncRes::s_minimumECSTTL = pdns_stou(*begin);
```

Comment:
`pdns_stou` can throw, which would abort the process I think.

Reply:
Nice catch!